.github/actions/dotnet/format/action.yml

```diff
@@ -8,16 +8,20 @@ inputs:
     required: false
     type: string
     default: 'bash'
+  CSHARPIER_VERSION:
+    required: false
+    type: string
+    default: ''
 
 runs:
   using: composite
   steps:
     - name: Install code formatter
       shell: ${{ inputs.SHELL }}
-      run: dotnet tool update --global csharpier
+      run: dotnet tool update --tool-path /tmp/dotnet-tools csharpier ${{ inputs.CSHARPIER_VERSION != '' && '--version' || '' }} ${{ inputs.CSHARPIER_VERSION }}
```

Comment:
Aggiunge sempre `inputs.CSHARPIER_VERSION`? Dovrebbe essere:

...` ${{ inputs.CSHARPIER_VERSION != '' && ('--version ' + inputs.CSHARPIER_VERSION) || '' }}`

No?

Reply:
Si, il risultato è lo stesso.

`${{ inputs.CSHARPIER_VERSION != '' && '--version' || '' }}` aggiunge `--version` solo se è stata impostata una versione di csharpier.

Dopo aggiungo la versione di csharpier che potrebbe essere stringa vuota oppure la versione stessa (in questo caso, preceduta da `--version`).

Avevo lasciato questa forma perché avevo il dubbio sulla sintassi github da usare per concatenare una stringa e un input all'interno di un'espressione.

La tua versione è comunque più chiara, faccio un test e se va pusho le modifiche 👍🏻

Grazie!

Reply:
Ah si certo la variabile anche se aggiunta sarà vuota  😅

Reply:
@jens-smeds-zupit Purtroppo l'operatore `+` non è supportato per il concat delle stringhe. Ho utilizzato `format`. Grazie per il tip, adesso la condizione è più chiara 👍🏻